src/static/menu-win.json

```diff
@@ -81,6 +81,15 @@
                     "separator": true
                 },
                 {
+                    "id": "EXPORT_UTILITY",
+                    "shortcut": {
+                        "keyChar": true,
+                        "modifiers": {
+                            "command": true
```

Comment:
Would you let me know if you have tested this on Windows? And, if so and if it does not throw an error, let me know I'll file a bug.